src/navigation/orbitalnavigator.cpp

```diff
@@ -1113,15 +1104,18 @@ void OrbitalNavigator::setAnchorNode(const SceneGraphNode* anchorNode,
     _syncedAnchorNode = anchorNode ? anchorNode->identifier() : "";
 
     // Need to reset velocities after the actual switch in anchor node,
-    // since the reset behavior depends on the anchor node.
-    if (changedAnchor && resetVelocitiesOnChange) {
+    // since the reset behavior depends on the anchor node
+    if (_resetVelocitiesOnAnchorChange) {
```

Comment:
Since there is a bool here that is stored between when the lua function call is made and the update happens, is there a risk that if you call two lua calls after each other, with different settings for the `_resetVelocitiesOnAnchorChange`, the bool will only take the last value? Or will this not be a problem? I assume that all the calls happen somewhere in a batch in the render loop

Reply:
Good question. Will do some testing

Reply:
I think it'll only make a difference if the Lua call is called in the same frame with different parameters and then the last one would could.

So:
_Frame i_
 1. Lua function is queued to reset velocties
 1. Lua function is queued to not reset velocities
 1. Navigation or rendering doesn't know about the Lua functions yet

_Frame i+1_
  1. Camera is updated
  1. Queued Lua functions from _i_ are applied and navigation is changed with not resetting velocities
  1. Rendering with probably(?) old state

_Frame i+2_
  1. ...
  1. Rendering with new state

Reply:
So the way it works is that once the retargeting and/or velocity reset has been triggered, it's irreversible. So when calling for example: 
```
openspace.navigation.setFocus("ISS", true, true);openspace.navigation.setFocus("Earth", false, false);
```
the "true, true" triggers the reset and retargeting, but the anchor node will be set to "Earth".  

I guess this COULD be considered unexpected behavior, but it's also a bit of a weird use case. We already before had some weird behaviors if setting the node while retargeting is being done, and setting the focus twice in the same frame is even more unexpected. 

So, maybe don't an issue, I think

For symmetry, not that

Reply:
Yea, I think even though it is unexpected behavior, I'd chuck it up to "people shouldn't do that" and it's fine to leave it